### ISSUE TITLE
add Auth 

### ISSUE THREAD

**Issue by theyij** [theyij/tokyo-dev-map#4]
- [ ] admin / admin

**Comment by brykka** [comment]
https://kentcdodds.com/blog/authentication-in-react-applications for further down the road

**Comment by brykka** [comment]
https://github.com/auth0/auth0.js what i am currently trying to use

**Comment by theyij** [comment]
This looks so difficult.. I'm play with this simple tutorial first.
https://auth0.com/blog/react-tutorial-building-and-securing-your-first-app/

**Comment by brykka** [comment]
yeah the routes aren’t ready exactly yet so i haven’t begun implementing auth0 yet. i’ll try tonight
